vendor/github.com/jacobsa/gcloud/gcs/read.go

```diff
@@ -49,12 +45,12 @@ func (b *bucket) NewReader(
 
 	// Switching to Go Storage Client Library.
 	if true {
-                rc, err = NewReaderSCL(ctx, req, b.name)
-                if err != nil {
-                        err = fmt.Errorf("Error in creating client through NewReaderSCL")
-                }
-                return
-        }
+		rc, err = NewReaderSCL(ctx, req, b.name, b.sclClient)
+		if err != nil {
+			err = fmt.Errorf("Error in creating client through NewReaderSCL")
```

Comment:
you can get because of different scenarios from newReaderSCL. You cannot simply say error in creating client for all types of errors. IMO there is no need to format every error we receive, we can return as is

Reply:
sure, will remove it.